themes/custom_theme/valencia-theme/src/templates/navigation.ftl

```diff
@@ -0,0 +1,69 @@
+<nav class="navbar navbar-default navbar-transparent navbar-fixed-top navbar-color-on-scroll ${nav_css_class}" id="navigation" role="navigation">
+	<div class="container">
+	<h1 class="hide-accessible"><@liferay.language key="navigation" /></h1>
+		<div class="navbar-header">
+			<button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#main-site-navigation">
+				<span class="sr-only">Menu</span>
```

Comment:
We should not hardcode languages as only english since other uses might be using a different language.  We should instead use a Liferay language key which will localize the text.  See line 3.